### DISCUSSION TITLE
2 PC's sync

### DISCUSSION THREAD

**Discussion by RACE4EVER** [mtkennerly/ludusavi#199] [Q&A] [answered]
I have 2 laptops and they have different profile Username ( one is `C:/Users/Asus `and other is `C/Users/User`). When i try to sync saves (via OneDrive) it creates 2 folders for each of them and replaces `mapping.yaml`.
Is there any solution except renaming user folder?

**Comment by mtkennerly** [comment, marked as answer]
You can use bidirectional redirects for this (on the "other" screen):

* On the `Asus` computer, make a bidirectional redirect from `C:/Users/Asus` to `C:/Users/main`
* On the `User` computer, make a bidirectional redirect from `C:/Users/User` to `C:/Users/main`

In the backups, you'll have a single `drive-C/Users/main` folder, but when you restore it, it will map back to the correct username.

**Reply by mtkennerly** [reply]
`Asus` is fine for the backups folder itself (the path configured on the backup screen). The only change you need to make is adding the redirects:

> <img width="637" alt="image" src="https://user-images.githubusercontent.com/4934192/235803773-8b786874-4681-4c04-bfa6-9ec2433931f1.png">

That way, you can tell Ludusavi:

* "I want my backups stored in `C:\Users\Asus\OneDrive\Ludusavi`" (because of the backup target)
* but also "inside of each individual backup, pretend that `C:\Users\Asus` is `C:\Users\main`" (because of the redirects)
